### PR TITLE
feat: Schedule and After-Event Video: Add video button to schedule

### LINKED ISSUE #5939: Schedule and After-Event Video: Add video button to schedule
For live video event rooms there is a button on the event schedule "Join Video". 
* This button should disappear after an event, but organizers can add a video of the event (already available in the session form options)
* If there is a video after the event available, please add a button "Watch Video" with a video icon in the same place. 
* Link to the specific session page - directly to the area of the video
* Please ensure this also works if there is a "Download Slides" button

Compare https://eventyay.com/e/cf877702/schedule

Screenshot from 2020-12-06 23-35-08

Compare https://eventyay.com/e/1f720482/schedule

Screenshot from 2020-12-06 23-27-25

### PR DESCRIPTION
**Pull request by akash2408**
Fixes #5939 

#### Short description of what this resolves:
onsch1
ontime1

notsch2
nottime2

join video button disappear after ending of the session if a video is available watch video button will appear

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/public/session-item.hbs [hunk 1]
```diff
@@ -16,11 +16,20 @@
               {{if this.slidesUploaded (t 'Download Slides') (t 'Link to Slides')}}
             </button>
           {{/if}}
-          {{#if @session.microlocation.hasVideoStream}}
-            <button class="ui basic {{text-color @session.track.color 'basic' 'inverted'}} button no-wrap" style={{css color=(text-color @session.track.color 'grey' 'lightgrey')}} {{action this.goToStream}}>
-              <i class="icon video"></i>
-              {{t 'Join Video'}}
-            </button>
+          {{#if this.sessionEnded}}
```
Comment (iamareebjamal) [inline app/components/public/session-item.hbs:19]: Opposite

**Comment by akash2408** [comment]
@iamareebjamal  please review this and let me know for any changes?

**Comment by iamareebjamal** [comment]
I have already reviewed, check my comment above

**Comment by akash2408** [comment]
Ohk, i already done above changes

**Comment by iamareebjamal** [comment]
No, otherwise it would show `Outdated` https://github.com/fossasia/open-event-frontend/pull/6014#discussion_r541771070

**Comment by iamareebjamal** [comment]
Update? Please finalize

File: app/components/public/session-item.js [hunk 2]
```diff
@@ -28,6 +29,17 @@ export default class SessionItem extends Component {
     return url.startsWith('https://open-event-api-dev.herokuapp.com') || url.startsWith('https://api.eventyay.com');
   }
 
+  get videoUploaded() {
+    const url = this.args.session.videoUrl;
+    return url.startsWith('https://open-event-api-dev.herokuapp.com') || url.startsWith('https://api.eventyay.com');
+  }
```
Comment (iamareebjamal) [inline app/components/public/session-item.js:35]: Why?

File: app/components/public/session-item.js [hunk 3]
```diff
@@ -28,6 +29,17 @@ export default class SessionItem extends Component {
...
+
```
Comment (iamareebjamal) [inline app/components/public/session-item.js:36]:
```suggestion
```

**Pull request merged**